### PR TITLE
Fixing Multi Tag Subscription Button

### LINKED ISSUE #5245: Multi-Tag Subscription Button
Finally, the multi-tag subscription button will be back, available for use, this issue intends to fix the popover problem
image
To:
image

The users can now subscribe to multiple tags in one go again!

I am submitting a PR for the same!

### PR DESCRIPTION
**Pull request by IshaGupta18**
Fixes #5245 
The multi-tag subscription now works!
image

### CODE REVIEW

**Comment by IshaGupta18** [comment]
@jywarren kindly review this PR if it's ready to be merged or not. Let me know what else can be done! Thanks a lot!

File: app/views/tag/_subscribe_button.html.erb [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
   .sub{
     border-radius: 5px;
     margin-bottom: 12px;
     padding: 5px;
   }
-
 </style>
+
+<script type="text/javascript">
+  $('#popover_button').popover({
```
Comment (jywarren) [inline app/views/tag/_subscribe_button.html.erb]: Wow this is a nice solution!!! More testable, too!

**Comment by jywarren** [comment]
Would you mind uploading a screen recording gif of the whole sequence, just so we can be extra sure, and also this will serve as a reference for future contributors for how it should work? Thank you!

Also, once we get system tests running, we'll be able to write a test for this :smile:

**Comment by IshaGupta18** [comment]
Absolutely, will do so soon! Thanks a lot!

**Comment by IshaGupta18** [comment]
Here you go!
sub

**Comment by IshaGupta18** [comment]
@jywarren is it good to go now?

**Comment by IshaGupta18** [comment]
@gauravano kindly have a look at this? If it's good to go?

**Comment by gauravano** [comment]
Hey @IshaGupta18, looks good to me. Although, I can see muti-tag subscribe button on dashboard but not on wiki sidebar or note sidebar. What conditions we are using to show them there?

Thanks!

**Comment by IshaGupta18** [comment]
Thanks a lot! The user will have to like that post to see the button.

**Comment by gauravano** [comment]
Hmm, why are we not showing that button to everyone?

**Comment by IshaGupta18** [comment]
I feel we should, but I was told to do this, but maybe we can remove that
condition and show it to everyone, it would encourage more involvement too.
What do you think?

**Comment by gauravano** [comment]
Also, if we are showing this button to node likers then I think we should add this button via AJAX when a user likes the post. Because, you don't know, if user is coming to that post again or not. So, showing that button with some kind of feature-discovery would definitely gain the user's interest. What do you think? 

This all not in this PR but for the future :smile:. Thanks @IshaGupta18 for great work on this!

**Comment by IshaGupta18** [comment]
We sure can do this! Would love to work on it, although we can get rid of
the condition too, whatever is best for it can be implemented!

**Comment by gauravano** [comment]
Let's see what @jywarren think?

**Comment by IshaGupta18** [comment]
Yes, then we can probably take it up in a follow up PR, because that would
be a little different from the aim of this PR. What do you think?

**Comment by gauravano** [comment]
Yeah, sure. Follow-up PR after a bit of discussion would be great! Thanks!

**Comment by jywarren** [comment]
Love this -- merging and will publish shortly. And yes, let's remove the condition!

**Pull request merged**